pilot/test/integration/testdata/app.yaml.tmpl

```diff
@@ -89,15 +89,20 @@ spec:
         - name: tcp-health-port
           containerPort: 3333
         livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 3333
+          exec:
+            command:
+            - /usr/local/bin/client
+            - -url
+            - http://localhost:3333/healthz
           initialDelaySeconds: 10
           periodSeconds: 10
           failureThreshold: 10
         readinessProbe:
-          tcpSocket:
-            port: tcp-health-port
+          exec:
+            command:
+            - /usr/local/bin/client
+            - -url
+            - http://localhost:3333/healthz
```

Comment:
why ? I put this in here to test that we are properly opening up ports in the envoy config for health checks, without inserting mixer filter in between. We need to test this even without auth.

Reply:
btw, liveness probes dont work with auth. Even without auth, they didn't work properly. I fixed the bugs and enabled this to ensure that probes continue to work in the non auth case.